orca-integrations-cloudfoundry/orca-integrations-cloudfoundry.gradle

```diff
@@ -0,0 +1,18 @@
+test {
+  useJUnitPlatform {
+    includeEngines "junit-vintage", "junit-jupiter"
+  }
+}
+
+dependencies {
+  compileOnly spinnaker.dependency("lombok")
+  annotationProcessor spinnaker.dependency("lombok")
+
+  compile project(":orca-core")
+
+  testCompile spinnaker.dependency("junitJupiterApi")
+  testCompile spinnaker.dependency("assertj")
+
+  testRuntime spinnaker.dependency("junitJupiterEngine")
+  testRuntime "org.junit.vintage:junit-vintage-engine:${spinnaker.version('jupiter')}"
```

Comment:
I'd strike vintage in any new module. It only exists where there are a mix of Spock/JUnit 5 tests in other modules.